pycycling/sterzo.py

```diff
@@ -22,10 +23,19 @@ async def enable_steering_measurement_notifications(self):
         await self._activate_steering_measurements()
 
     async def _activate_steering_measurements(self):
-        with importlib.resources.files(__package__).joinpath('data/sterzo-challenge-codes.dat').open('rb') as fp:
-            fp.seek(self._latest_challenge * 2, 1)
-            code_1 = int.from_bytes(fp.read(1), 'little')
-            code_2 = int.from_bytes(fp.read(1), 'little')
+        # moved .dat file out of 'data' because accessing directories not possible for importlib.resources.path
```

Comment:
I think we can keep the data directory, we just need to add `__init__.py` to make it a Python package

Reply:
Do you mean putting a `__init__.py` to the `data` directory like so?
```
├── pycycling
       ├── sterzo.py
       ├── data
       │   ├── __init__.py
       │   └── sterzo-challenge-codes.dat
```

Reply:
Yep! Can be completely empty I believe.